inst/qml/common/classical/SimpleMainEffects.qml

```diff
@@ -22,15 +22,15 @@ import JASP.Controls
 
 Section
 {
-	title:	qsTr("Simple Main Effects")
+	title:	qsTr("Simple Main Effects"); info: qsTr("The simple main effects represent the effect of one independent variable for each level of the other independent variable, by conducting an ANOVA for each subset of the data as specified by the moderator variables.")
 	property alias source:	availableTerms.source
 
 	VariablesForm
 	{
 		preferredHeight:	170 * preferencesModel.uiScale
-		AvailableVariablesList	{ name: "simpleMainEffectAvailableFactors";		title: qsTr("Factors")				; id: availableTerms }
-		AssignedVariablesList	{ name: "simpleMainEffectFactor";				title: qsTr("Simple Effect Factor") ; singleVariable: true }
-		AssignedVariablesList	{ name: "simpleMainEffectModeratorFactorOne";	title: qsTr("Moderator Factor 1")	; singleVariable: true }
-		AssignedVariablesList	{ name: "simpleMainEffectModeratorFactorTwo";	title: qsTr("Moderator Factor 2")	; singleVariable: true }
+		AvailableVariablesList	{ name: "simpleMainEffectAvailableFactors";		title: qsTr("Factors")	; info: qsTr("This box contains all the independent variables included in the analysis.")			; id: availableTerms }
+		AssignedVariablesList	{ name: "simpleMainEffectFactor";				title: qsTr("Simple Effect Factor") ; info: qsTr("In this box, select the independent variable to determine the effect of this variable, conditional on the levels of the moderator factor(s).") ; singleVariable: true }
+		AssignedVariablesList	{ name: "simpleMainEffectModeratorFactorOne";	title: qsTr("Moderator Factor 1")	; info: qsTr(" In this box, select the independent variable that will represent the different levels.") ;singleVariable: true }
+		AssignedVariablesList	{ name: "simpleMainEffectModeratorFactorTwo";	title: qsTr("Moderator Factor 2")	; info: qsTr(" In this box, selector an optional, additional independent variable.") ;singleVariable: true }
```

Comment:
selector -> select
-> "Select an optional, additional independent variable to use as a second moderator."

Reply:
In general, you could remove "in this box" to make it consistent with the rest of the texts